jacodb-analysis/src/main/kotlin/org/jacodb/analysis/codegen/Generator.kt

```diff
@@ -158,4 +158,13 @@ fun main(args: Array<String>) {
     }
 
     codeRepresentation.dumpTo(projectPath)
-}
\ No newline at end of file
+    try {
+        val processBuilder = ProcessBuilder()
+        processBuilder.directory(File("generated\\UtBotTemplateForIfdsSyntheticTests".replace('\\', File.separatorChar)))
+        processBuilder.command("/bin/bash", "./gradlew", "assemble")
```

Comment:
you dont need bash here, gradlew is enough. with bash this wont work on linux

Reply:
1. It works on my linux (Ubuntu 22.04.1)
2. If I remove ```/bin/bash```, the code in idea will fail with the following error. Is this a problem in my environment?
```
java.io.IOException: Cannot run program "./gradlew" (in directory "generated/UtBotTemplateForIfdsSyntheticTests"): error=13, Permission denied
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1142)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1073)
	at org.jacodb.analysis.codegen.GeneratorKt.main(Generator.kt:166)
Caused by: java.io.IOException: error=13, Permission denied
	at java.base/java.lang.ProcessImpl.forkAndExec(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:313)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:244)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1109)
	... 2 more
```